manifests/router.pp

```diff
@@ -21,16 +21,16 @@
 # @param service_ensure
 #   Specify if qdrouterd should be running or stopped.
 class qpid::router(
-  $router_id               = $qpid::router::params::router_id,
-  $mode                    = $qpid::router::params::router_mode,
-  $config_file             = $qpid::router::params::config_file,
-  $router_packages         = $qpid::router::params::router_packages,
-  $open_file_limit         = $qpid::router::params::open_file_limit,
-  $worker_threads          = $qpid::router::params::worker_threads,
-  $hello_interval          = $qpid::router::params::hello_interval,
-  $hello_max_age           = $qpid::router::params::hello_max_age,
-  $service_ensure          = true,
-  $service_enable          = undef,
+  Stdlib::Host $router_id = $qpid::router::params::router_id,
```

Comment:
I don't know how this is limited. Can it be a FQDN or an IP or just any string?

Reply:
> The unique identifier of the router. This ID is used as the container-id (container name) at the AMQP protocol level. If it is not specified, the router shall generate a random identifier at startup.

That reads like it could be any ole string.